client/src/main/java/io/quarkus/vault/client/VaultClientConfiguration.java

```diff
@@ -0,0 +1,11 @@
+package io.quarkus.vault.client;
+
+import java.time.InstantSource;
+
+public class VaultClientConfiguration {
```

Comment:
FYI never used

Reply:
Missed. This was replaced with the VaultClient.Builder. Removing.